src/client/src/models/Scene.js

```diff
@@ -56,6 +56,12 @@ const Scene = {
   save: function() {
     const list = Scene.getScenes()
 
+    // this is a check to make sure the value entered into start is a number
+    const start_check = Number(Scene.current.start)
+    if (isNaN(start_check)) {
```

Comment:
Can we have additional error handling around creating a scene with a start value that is the same as one that already exists and possibly an error in case the start time is not specified as it breaks the notion of being able to order scenes by using the start value.

Reply:
I think we should order them first, it would make it easier to see if one already exists.